### ISSUE TITLE
In-Game: [YYC] audio_stop_sound() crashes with OpenAL Error if stopping multiple sounds in 2024.800 Betas

### ISSUE THREAD

**Issue by shichen85** [YoYoGames/GameMaker-Bugs#7200]
### Description
The sample plays 4 audio files every step and calls `audio_stop_sound()` every 5 steps:

```
   //Step
	audio_play_sound(snd_1, 0, false);
	audio_play_sound(snd_2, 0, false);
	audio_play_sound(snd_3, 0, false);
	audio_play_sound(snd_4, 0, false);
   //alarm0
  audio_stop_sound(snd_1);
  audio_stop_sound(snd_2);
  audio_stop_sound(snd_3);
  audio_stop_sound(snd_4);
  alarm[0] = 5
```

When exporting for Windows YYC, the output shows `OpenAL Error: 40963 (Invalid value)` error and eventually crashes with exit code  -1073741819.

This does not happen to the 600 runtime.

### Steps To Reproduce
1. Run the sample for YYC export in runtime 2024.800.0.628, see OpenAL errors in output and crash
2. Repeat in runtime 2024.600.0.607 and see it does not crash

### Which version of GameMaker are you reporting this issue for?
IDE v2024.800.0.605 Runtime v2024.800.0.628

### Which operating system(s) are you seeing the problem on?
Windows 10.0.22000.0

### Which platform(s) are you seeing the problem on?
Windows

[e2046ad6-98a5-4872-a2b1-311affeae635](https://api.gamemaker.io/api/github/downloads/e2046ad6-98a5-4872-a2b1-311affeae635)

**Comment by shichen85** [comment]
The crash might be related to the changes for https://github.com/YoYoGames/GameMaker-Bugs/issues/6727#issuecomment-2258103297

**Comment by DragoniteSpam** [comment]
I've been getting this error code on and off for a few betas now, but I haven't been able to narrow down what's causing it. On VM it doesn't outright crash, but the audio playback for certain sounds ends up sounding like a skipping record.

**Comment by shichen85** [comment]
Yes, we experience the audio not properly playing in YYC after a few calls of `audio_stop_sound()`, which eventually leads to a crash. Here is the sample that I submitted privately that should consistently crash.
[gmb-6727-xasd (2).zip](https://github.com/user-attachments/files/16577098/gmb-6727-xasd.2.zip)

**Comment by YYDan** [comment]
Possibly due to https://github.com/YoYoGames/GameMaker-Bugs/issues/7026 also

**Comment by toby-yoyo** [comment]
Fixed an issue where a sound ending playback normally while being stopped could make the conditions for its voice recycling invalid.

**Comment by scott-dunbar** [comment]
Verified fixed in Beta IDE v2024.800.0.613 Beta Runtime v2024.800.0.636
